src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -1322,6 +1322,15 @@ def load_arguments(self, _):
                        'going to be replicated to. This property is updatable. Expected value: '
                        'json-string/json-file/@json-file.')
             c.argument('default_file_link', help='The default configuration link of the artifact, must be a readable storage page blob.')
+
+    with self.argument_context('sig gallery-application create') as c:
+        c.argument('package_file_name', help='The name to assign the downloaded package file on the VM. This is limited to 4096 characters.' 
+                                             'If not specified, the package file will be named the same as the Gallery Application name.')
+        c.argument('config_file_name', help='The name to assign the downloaded config file on the VM. This is limited to 4096 characters. '
+                                            'If not specified, the config file will be named the Gallery Application name appended with "_config"')
```

Comment:
> If not specified, the package file will be named the same as the Gallery Application name.
> If not specified, the config file will be named the Gallery Application name appended with "_config"

May I ask are these default names handled by the service side?

Reply:
I see, so CLI does not need to add additional default value processing logic, right?

Reply:
No, it does not